CompetitionTemplate/src/competition/subsystems/autonomous/selection/SetupLowBarCommand.java

```diff
@@ -0,0 +1,19 @@
+package competition.subsystems.autonomous.selection;
+
+import xbot.common.properties.XPropertyManager;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.autonomous.RaiseArmAndTraverseDefenseCommandGroup;
+
+
+public class SetupLowBarCommand extends SetupRaiseArmAndTraverseCommand {
+    
+    final String label = "LowBarAuto ";
```

Comment:
nitpicking - these labels shouldn't have spaces, the base class should be in charge of the delimiter (whether that be a space or dash or whatever)